inst/rmarkdown/templates/lipics/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,302 @@
+  :::
+  
+  ::: {.exercise data-latex="\label{testenv-exercise}"}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+  
+  ::: {.definition data-latex="\label{testenv-definition}"}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+  
+  ::: {.example data-latex="\label{testenv-example}"}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+  
+  ::: {.note data-latex="\label{testenv-note}"}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+  
+  \begin{note*}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  \end{note*}
+  
+  ::: {.remark data-latex="\label{testenv-remark}"}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+  
+  \begin{remark*}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  \end{remark*}
+  
+  ::: {.claim data-latex="\label{testenv-claim}"}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+  
+  
+  \begin{claim*}\label{testenv-claim2}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  \end{claim*}
+  
+  ::: {.proof data-latex=""}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+  
+  ::: {.claimproof data-latex=""}
+  Fusce eu leo nisi. Cras eget orci neque, eleifend dapibus felis. Duis et leo dui. Nam vulputate,   velit et laoreet porttitor, quam arcu facilisis dui, sed malesuada risus massa sit amet neque.
+  :::
+output:
+  rticles::lipics_article
+  #bookdown::pdf_book:
+  #  base_format: rticles::lipics_article # for using bookdown features like \@ref()
+  #  extra_dependencies: plainurl.bst
```

Comment:
```suggestion
output:
  rticles::lipics_article: default
  bookdown::pdf_book:
    base_format: rticles::lipics_article # for using bookdown features like \@ref()
```

* `plainurl.bst` is not needed
* You can remove the comment because a Rmarkdown document can have multiple output defined.